src/WebUI/Selection/SelectionActionCreator.ts

```diff
@@ -40,14 +40,16 @@ export class SelectionActionsCreator extends ActionCreatorBase {
                 if (!!value && typeof value !== "object" && typeof value !== "function") {
                     routeValues[k] = value;
                 }
-
-                // We will send undefined or empty for values that we want deleted from the url
-                if (!value) {
-                    delete routeValues[k];
-                }
             });
         }
 
+        Object.keys(routeValues).forEach(k => {
+            // delete the properties which has empty value
+            if (!routeValues[k]) {

```

Comment:
"false" might be removed in this case. I don't think we have true/false anywhere, so we are fine at the moment